recipe/meta.yaml

```diff
@@ -20,9 +20,10 @@ requirements:
   host:
     - python >=3.7
     - pip
+    - poetry-core >=1.0.0
   run:
     - python >=3.7
-    - attrs >=20.1.0
+    - attrs
```

Comment:
There's no constraint upstream:
https://github.com/Tinche/cattrs/blob/v1.6.0/pyproject.toml#L27
Perhaps this was loosened since the last release (and the switch to `poetry`).